loadtest/scripts/validator_failures.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import json
+import random
+import subprocess
+import time
+
+
+def failure_restarts(validators, ssh_key, num_concurrent):
+    """
+    Restarts multiple validators at a time. Randomly sleeps between 1 to 5 mins
+    to allow the nodes come up.
+    """
+    i = 0
+    # Loop through all validators
+    while i < len(validators):
+        j = 0
+        # Restart num_concurrent validators
+        while j < num_concurrent:
+            print(f"Restarting validator {validators[i + j]}")
+            subprocess.check_output(
+                f"ssh -i {ssh_key} ubuntu@{validators[i + j]} 'sudo -S -p \"\" systemctl restart seid'")
+            j += 1
+        i += num_concurrent
+
+        time.sleep(random.randint(60, 300))
+
+
+def delegation_change(chain_id):
+    """
+    Changes delegation of validators to create imbalanced delegation. Delegates
+    10 * total to the top 4 validators for a period of time between 10 minutes
+    to 1 hour.
+    """
+    # First get total voting power
+    admin_acc = _get_admin_acc()
+    seid_query_cmd = f"seid q staking delegations {admin_acc} --chain-id {chain_id} --output json"
+    staking_output = json.loads(
+        subprocess.check_output([seid_query_cmd], stderr=subprocess.STDOUT,
+                                shell=True))
+    total_delegation = 0
+    validator_addrs = []
+    for resp in staking_output["delegation_responses"]:
+        total_delegation += int(resp["balance"]["amount"])
+        validator_addrs.append(resp["delegation"]["validator_address"])
+    # Delegate 20x to 1/4 of the validators
+    delegations = {}
+    for i in range(len(validator_addrs // 4)):
+        delegations[validator_addrs[i]] = 10 * total_delegation
+        print(f"Delegating {10 * total_delegation} to {validator_addrs[i]}")
+        seid_staking_cmd = f"printf '12345678\n' seid tx staking delegate {validator_addrs[i]} {10 * total_delegation}usei --from admin --chain-id {chain_id} -b block -y"
```

Comment:
not a blocking nit but could we make this a helper or something that just printf to  any seid command?